docker-compose.yml

```diff
@@ -108,6 +108,7 @@ services:
       - _APP_FUNCTIONS_CPUS
       - _APP_FUNCTIONS_MEMORY
       - _APP_FUNCTIONS_MEMORY_SWAP
+      - _APP_FUNCTIONS_ENVS
```

Comment:
We should add this on `compose.phtml` as well

Reply:
Will do 👍🏻